pkg/kubernetes/annotation.go

```diff
@@ -13,6 +13,7 @@ const (
 	AnnotationSpinnakerArtifactType       = `artifact.spinnaker.io/type`
 	AnnotationSpinnakerMonikerApplication = `moniker.spinnaker.io/application`
 	AnnotationSpinnakerMonikerCluster     = `moniker.spinnaker.io/cluster`
+	AnnotationSpinnakerVersioned          = `strategy.spinnaker.io/versioned`
```

Comment:
To align with the rest of the constants, can you rename this const to `AnnotationSpinnakerStrategyVersioned`?